src/BuildCheck.UnitTests/EditorConfigParser_Tests.cs

```diff
@@ -0,0 +1,125 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Text;
```

Comment:
super-nit: Several of these namespace imports look unnecessary. (Comment applies to other test source files as well. Perfectly fine to keep it this way if they are verbatim copies from Roslyn, though.)

Reply:
Agree imports were not needed: IDE not reported this as well as the build itself. 
Reported improvement : https://github.com/dotnet/msbuild/issues/10156.